cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -117,7 +117,7 @@ macro (build_addon target prefix libs)
                 if("${matchres}" EQUAL 0)
                   string(REPLACE " " ";" loop_var "${loop_var}")
                   list(GET loop_var 1 include_name)
-                  string(REGEX REPLACE "[<>\"]" "" include_name "${include_name}")
+                  string(REGEX REPLACE "[<>\"]|kodi/" "" include_name "${include_name}")
```

Comment:
to clarify - if the "kodi" folder is part of the file tree from add-ons and is not dynamically generated/referenced from the app name in some other place, hardcoding it is fine. But as I have no idea of cmake I thought I just mention it because of the DevCon discussion we had. I didn't mean to block merging this PR.